data-prepper-core/src/main/java/org/opensearch/dataprepper/pipeline/server/ShutdownHandler.java

```diff
@@ -22,6 +25,9 @@ public class ShutdownHandler implements HttpHandler {
     private final DataPrepper dataPrepper;
     private static final Logger LOG = LoggerFactory.getLogger(ShutdownHandler.class);
 
+    static final Path SHUTDOWN_FILE_PATH = Path.of("Is-Shutdown.log");
```

Comment:
What should the path here be? I noticed that when running in docker the data prepper directory structure had a `log/data-prepper/` directory which could make sense, but I am not sure how to make it consistent for running locally and with docker.